packit_service/models.py

```diff
@@ -1970,6 +1970,8 @@ class CoprBuildTargetModel(GroupAndTargetModelConnector, Base):
         "CoprBuildGroupModel", back_populates="copr_build_targets"
     )
 
+    scan = relationship("ScanModel", back_populates="copr_build_target")
```

Comment:
I was wondering whether this should not be connected on the `PipelineModel` level. Have you considered this as well?

Reply:
We agreed on keeping the actual design.